skimage/restoration/_rolling_ball.py

```diff
@@ -27,13 +38,16 @@ def rolling_ball(image, *, radius=100, kernel=None, nansafe=False, num_threads=N
     nansafe: bool, optional
         If ``False`` (default), the function assumes that none of the values
         in `image` are ``np.nan``, and uses a faster implementation.
-    num_threads: int, optional
-        The maximum number of threads to use. If ``None``, the function uses
-        the OpenMP default value; typically, it is equal to the maximum number
-        of virtual cores.
+    num_workers: int, optional
+        The maximum number of threads to use. If ``None``, use the OpenMP
+        default value; typically equal to the maximum number of virtual cores.
         Note: This is an upper limit to the number of threads. The exact number
         is determined by the system's OpenMP library.
 
+        .. versionchanged:: 0.23
+
+            Replaced old parameter `num_threads` in 0.23.
```

Comment:
```suggestion
        .. versionadded:: 0.26
            Replaces deprecated parameter `num_threads`.
```
Based on recently merged #7739?